### DISCUSSION TITLE
Overdue in Agenda view

### DISCUSSION THREAD

**Discussion by DHS1001** [orgzly-revived/orgzly-android-revived#168] [General]
I started to use the application from last week, and found the overdue section in Agenda is a bit hard to understand than it should.
Eg. A deadline is set to 12/02 (Date) for a task, and finished time is 12/02 13:30 (Date and time).
My understanding is, in this case, this is not considered as overdue, because it is finished on the date.
But in `ad. 7`, it will showed this task on the top as overdue.
Just want to see if this is by design? or it is misused/misunderstanding here? 
Thanks

**Comment by amberin** [comment]
Hi! Per the [search syntax documentation](https://www.orgzly.com/docs#search):

> You can add ad.DAYS to any query to display search results grouped by day.

You need to combine the ad query with another query to search for items based on state. For example: `ad.7 .it.done`. With your current query, you are asking to show everything and group it by date, cutting off at 7 days into the future.

**Reply by DHS1001** [reply]
@amberin Thanks for the hints, it makes sense now.
